dev/com.ibm.ws.jsp.2.3_fat/publish/servers/jspJava7Server/server.xml

```diff
@@ -0,0 +1,24 @@
+<!--
+    Copyright (c) 2024 IBM Corporation and others.
+    All rights reserved. This program and the accompanying materials
+    are made available under the terms of the Eclipse Public License 2.0
+    which accompanies this distribution, and is available at
+    http://www.eclipse.org/legal/epl-2.0/
+    
+    SPDX-License-Identifier: EPL-2.0
+ -->
+<server description="Server for Java 7 Source Option in JSP">
+
+    <include location="../fatTestPorts.xml"/>
+
+    <featureManager>
+        <feature>jsp-2.3</feature>
+        <feature>localConnector-1.0</feature>
+    </featureManager>
+
+    <!-- Note: jdkSourceLevel has been marked internal -->
+    <jspEngine jdkSourceLevel="17" prepareJSPs="0"/>
+
+    <logging traceSpecification="*=info=enabled:com.ibm.ws.jsp*=all" maxFileSize="32" maxFiles="24" traceFormat="BASIC"/>
```

Comment:
We usually move the tracing to the bootstrap.properties.

Reply:
I see it is already there, but you'll want to update the other attributes you have defined here.

Reply:
I removed the server.xml logging and will rely on the bootstrap.